uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/AbstractDownloadControllerIT.java

```diff
@@ -492,23 +550,21 @@ void submitJobWithIncludeIsoformFlag() throws Exception {
         Assertions.assertTrue(Files.exists(unzippedFile));
         String resultsJson = Files.readString(unzippedFile);
         List<String> primaryAccessions = JsonPath.read(resultsJson, "$.results.*.primaryAccession");
-        Assertions.assertEquals(12, primaryAccessions.size());
+        Assertions.assertEquals(14, primaryAccessions.size());
         Assertions.assertTrue(primaryAccessions.contains("P00001"));
         Assertions.assertTrue(primaryAccessions.contains("P00011-2"));
         Assertions.assertTrue(primaryAccessions.contains("P00012-2"));
     }
 
     @Test
-    void submitJob_H5_Format_Star_Query_Success() throws Exception {
+    void submitJob_H5_Format_Star_Query_Aborted() throws Exception {
```

Comment:
Do we need to add another test for success? Or there is any other test covering it?

Reply:
there is already test case for success case.

Reply:
Ok. i checked again, it was not there.. added the success case.